lib/classes/PluginManager.js

```diff
@@ -352,6 +353,13 @@ class PluginManager {
     });
   }
 
+  assignDefaultOptions(command) {
+    _.forEach(command.options, (value, key) => {
+      if (value.default && (!this.cliOptions[key] || this.cliOptions[key] === true)) {
```

Comment:
It was in options validation and I wanted them to be compatible :p
https://github.com/rpunkfu/serverless/blob/d9dcd73e5daf61302610d739e89ecece1a382552/lib/classes/PluginManager.js#L287